### ISSUE TITLE
Error starting containers with bosco and latest docker native for mac

### ISSUE THREAD

**Issue by bettiolo** [tes/bosco#154]
`bosco run` returns me the following error:

```
[05:16:37] Bosco: Starting infra-nginx-gateway: docker-registry.tescloud.com/tescloud/infra-nginx-gateway:latest...
[05:16:37] Bosco: Failed to start Docker image: HTTP code is 400 which indicates error: undefined - [object Object]
[05:16:37] Bosco: Error: HTTP code is 400 which indicates error: undefined - [object Object]
```

**Comment by bettiolo** [comment]
Fixing docker-modem: https://github.com/apocas/docker-modem/pull/64

**Comment by bettiolo** [comment]
The real error is: `starting container with HostConfig was deprecated since v1.10 and removed in v1.12`

**Comment by bettiolo** [comment]
Deprecation: https://docs.docker.com/v1.11/engine/reference/api/docker_remote_api_v1.23/#start-a-container

**Comment by cliftonc** [comment]
Published with 0.5.45
